bot/bot.py

```diff
@@ -79,8 +92,10 @@ def feedback(message):
                          reply_markup=Main_mark_up)
         return
     time_at_now = time.strftime("%H:%M:%S %Y.%m.%d", time.localtime())
-    form = '''Feedback from {0} - @{1} ({2});\nDate: {3};\nText: {4}'''.format(message.from_user.first_name, message.from_user.username, message.from_user.id, time_at_now,
-                        message.text)
+    form = '''Feedback from {0} - @{1} ({2});\nDate: {3};\nText: {4}'''.format(message.from_user.first_name,
+                                                                               message.from_user.username,
+                                                                               message.from_user.id, time_at_now,
```

Comment:
Лучше тоже разнести по разным строкам:
```suggestion
                                                                               message.from_user.id,
                                                                               time_at_now,
```